sources/asset_manager.move

```diff
@@ -298,11 +309,9 @@ module balanced::asset_manager{
                 withdraw_to::amount(&message),
                 ctx
             );
-            xcall::execute_call_result(xcall,ticket,true,fee,ctx);
-        }else{
-            xcall::execute_call_result(xcall,ticket,false,fee,ctx);
         };
-        
+
+        xcall::execute_call_result(xcall,ticket,true,fee,ctx);
     }
```

Comment:
we have to revert when type send in message and type in type arguments do not match  , but in these cases 
`   assert!(
            verified,
            ProtocolMismatch
        );

        let method: vector<u8> = deposit::get_method(&msg);
        assert!(
            method == WITHDRAW_TO_NAME || method == WITHDRAW_NATIVE_TO_NAME,
            UnknownMessageType
        );
        `   we can actually trigger rollback.

Reply:
Can probably be the same for all contracts, revert if method does not exists or in case of protocol mismatch